MaxText/layers/normalizations.py

```diff
@@ -47,3 +47,46 @@ def __call__(self, x: jnp.ndarray) -> jnp.ndarray:
 
     scale = jnp.asarray(scale, self.dtype)
     return y * scale
+
+
+class LayerNorm(nn.Module):
```

Comment:
Honestly throw this into yoru GPT3 module? I think it is GPT3 specific at this point?

Reply:
And call it GPT3LayerNorm

Reply:
Good idea.

Reply:
Done.